File: Python/Estrutura repetitiva/quadrante.py
# Entrada de dados
x = int(input("Digite o valor de X: "))
y = int(input("Digite o valor de Y: "))

# Estrutura condicional
if (x != 0 and y != 0):
    # Estrutura repetitiva
    while (x != 0 and y != 0):
        # Estrutura condicional
        if x > 0 and y > 0:
            print("Suas coordendas pertencem ao primeiro quadrante (Q1).")
        elif x < 0 and y > 0:
            print("Suas coordendas pertencem ao segundo quadrante (Q2).")
        elif x < 0 and y < 0:
            print("Suas coordendas pertencem ao terceiro quadrante (Q3).")
        else:
            print("Suas coordendas pertencem ao quarto quadrante (Q4).")
        x = int(input("Digite o valor de X: "))
        y = int(input("Digite o valor de Y: "))

# Saída de dado
print("O programa foi terminado!")

File: C/Estrutura repetitiva/quadrante.c
// Biblioteca
#include <stdio.h>

int main()
{
    // Variável
    int x, y;

    // Entrada de dados
    printf("Digite o valor de X: ");
    scanf("%d", &x);

    printf("Digite o valor de Y: ");
    scanf("%d", &y);

    // Estrutura condicional
    if (x != 0 && y != 0) {
        // Estrutura repetitiva
        while (x != 0 && y != 0) {
            // Estrutura condicional
            if (x > 0 && y > 0) {
                printf("Suas coordenadas pertencem ao primeiro quadrante (Q1).\n");
            }
            else if (x < 0 && y > 0) {
                printf("Suas coordenadas pertencem ao segundo quadrante (Q2).\n");
            }
            else if (x < 0 && y < 0) {
                printf("Suas coordenadas pertencem ao terceiro quadrante (Q3).\n");
            }
            else {
                printf("Suas coordenadas pertencem ao quarto quadrante (Q4).\n");
            }

            // Adiciona uma linha em branco antes de solicitar novos valores
            printf("\nDigite o valor de X: ");
            scanf("%d", &x);
            printf("Digite o valor de Y: ");
            scanf("%d", &y);
        }
    }

    // Saída de dados
    printf("O programa foi terminado!");

    return 0;
}
